assets/chezmoi.io/docs/developer-guide/contributing-changes.md

```diff
@@ -19,7 +19,13 @@ that:
 * The documentation is updated, if necessary. For new features you should add an
   entry in `assets/chezmoi.io/docs/user-guide/` and a complete description in
   `assets/chezmoi.io/docs/reference/`. See the [website][website] for
-  instructions on how to build and view a local version of the documentation.
+  instructions on how to build and view a local version of the documentation. By
+  default, chezmoi will panic if a flag is undocumented or a long help is
+  missing for a command. You can disable this panic during development by
+  setting the environment variable `CHEZMOIDEV` to `ignoreflags=1,ignorehelp=1`.
+  Once you have documented the command and its flags, run
+  `CHEZMOIDEV=ignoreflags=1,ignorehelp=1 go generate` to generate the embedded
+  documentation.
```

Comment:
Can we add a `make` target that does this? `make generate` or `make embed`?

Reply:
Good idea. Done.